src/HierarchyLogEntry.h

```diff
@@ -38,20 +39,19 @@ class HierarchyLogEntry
     void notePeerWrite();
 
     /// Start recording total time spent communicating with peers
-    void startPeerClock();
+    void startPeerClock() { peeringTime_.resume(); }
     /**
      * Record total time spent communicating with peers
      * \param force whether to overwrite old recorded value if any
      */
```

Comment:
This documentation is very stale and must be fixed.

```suggestion
    /// Pause recording after a startPeering() call.
```

Reply:
Fixed.